scripts/download_glue_data.py

```diff
@@ -49,8 +49,26 @@ def download_and_extract(task, data_dir):
     with zipfile.ZipFile(data_file) as zip_ref:
         zip_ref.extractall(data_dir)
     os.remove(data_file)
+    if task == "SNLI":
```

Comment:
This should be its own PR—we're not dealing with CoLA here.

Reply:
This wasn't me. I forked from load_tsv branch, and it was there. I presume this diff won't exist once load_tsv is merged?

Reply:
That already was merged (https://github.com/jsalt18-sentence-repl/jiant/pull/477), so something is wrong here. Make sure you don't have any conflicts with master.